api/v1alpha/README.md

```diff
@@ -321,6 +321,7 @@ Class | Method | HTTP request | Description
  - [AWSAutoScaling](docs/AWSAutoScaling.md)
  - [AWSCloudProviderContainer](docs/AWSCloudProviderContainer.md)
  - [AWSCloudProviderContainerAllOf](docs/AWSCloudProviderContainerAllOf.md)
+ - [AWSCloudProviderContainerAllOf1](docs/AWSCloudProviderContainerAllOf1.md)
```

Comment:
I remember we wanted to run a transformation to remove AWSCloudProviderContainerAllOf. Do we know why we have a new object introduced?